nat-lab/bin/cleanup_natlab_processes

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "Executing natlab process cleanup script"
+
+for pid in $(ps -e -o pid=); do
+    # Skip non-testing processes
+    if ! grep --null-data --text KILL_ID /proc/${pid}/environ; then
+        continue
+    fi
+
+    # KILL
+    cmd=$(cat /proc/${pid}/cmdline | sed 's/\x00//g' || echo "N/A")
```

Comment:
just 
sed $pattern $file

Reply:
And `sed 's/\x00//g' $file` is an overkill when `tr -d '\000' < $file` is enough ;)

Reply:
fixed